packages/amplify-graphql-index-transformer/src/resolvers/generators/vtl-generator.ts

```diff
@@ -0,0 +1,10 @@
+import {
+  TransformerContextProvider,
+  TransformerResolverProvider
+} from '@aws-amplify/graphql-transformer-interfaces';
+
+import { PrimaryKeyDirectiveConfiguration } from '../../types';
+
+export interface PrimaryKeyVTLGenerator {
+  generate(config: PrimaryKeyDirectiveConfiguration, ctx: TransformerContextProvider, resolverMap: Map<TransformerResolverProvider, string>): void;
```

Comment:
Index directive lives in the same package, so we need to name the method more specific to primaryKey.

Reply:
Update to `generatePrimaryKeyInfo`